src/Sulu/Component/Rest/ListBuilder/ListRestHelper.php

```diff
@@ -139,8 +146,46 @@ public function getSearchPattern()
      */
     public function getSearchFields()
     {
-        $searchFields = $this->getRequest()->get('searchFields');
+        $searchFields = $this->getRequest()->get(self::PARAMETER_SEARCH_FIELDS);
 
         return ($searchFields != null) ? explode(',', $searchFields) : [];
     }
+
+    /**
+     * Returns an array of available filters.
+     *
+     * @return string[]
+     */
+    public function getFilters()
+    {
+        $filters = [];
+
+        $parameters = $this->getRequest()->query->all();
+
+        foreach ($parameters as $name => $value) {
+            // ignore empty values and none filter parameters
+            if ($value && !in_array($name, $this->getNoneFilterParameters())) {
+                $filters[$name] = $value;
```

Comment:
Not sure if it is a good idea to handle this automatically... What if somebody added an own parameter to his controller, and still uses the `ListRestHelper` afterwards? Then this own parameter would be passed here, although there is no `FieldDescriptor` for it, which would probably cause an error which is hard to find...